opm/parser/eclipse/EclipseState/Tables/MsfnTable.hpp

```diff
@@ -21,60 +21,33 @@
 #define	OPM_PARSER_MSFN_TABLE_HPP
 
 #include "SimpleTable.hpp"
+#include <opm/parser/eclipse/EclipseState/Tables/TableEnums.hpp>
 
 namespace Opm {
-    // forward declaration
-    class TableManager;
-
     class MsfnTable : public SimpleTable {
     public:
-
-        friend class TableManager;
-        MsfnTable() = default;
-
-        /*!
-         * \brief Read the PMISC keyword and provide some convenience
-         *        methods for it.
-         */
-        void init(Opm::DeckItemConstPtr item)
+        MsfnTable(Opm::DeckItemConstPtr item)
         {
-            SimpleTable::init(item,
-                             std::vector<std::string>{
-                                 "GasPhaseFraction",
-                                 "GasSolventRelpermMultiplier",
-                                 "OilRelpermMultiplier"
-                                     });
-
-            SimpleTable::checkNonDefaultable("GasPhaseFraction");
-            SimpleTable::checkMonotonic("GasPhaseFraction", /*isAscending=*/true);
-            SimpleTable::assertUnitRange("GasPhaseFraction");
-
-            SimpleTable::checkNonDefaultable("GasSolventRelpermMultiplier");
-            SimpleTable::checkMonotonic("GasSolventRelpermMultiplier", /*isAscending=*/true, /*isStriclyMonotonic=*/false);
-
-            SimpleTable::checkNonDefaultable("OilRelpermMultiplier");
-            SimpleTable::checkMonotonic("OilRelpermMultiplier", /*isAscending=*/false, /*isStriclyMonotonic=*/false);
-
+            m_schema = std::make_shared<TableSchema>( );
+            m_schema->addColumn( ColumnSchema( "GasPhaseFraction", Table::STRICTLY_INCREASING  , Table::DEFAULT_NONE));
```

Comment:
the column names of the other tables use 'ALL_UPPERCASE' instead if 'CamelCase'. I know, that was there before, but it would be a good time to fix it IMO.